libs/Helpers.py

```diff
@@ -23,3 +23,12 @@ def gte(self, version, target):
         except ValueError:
             # Returning False on exception as a workaround for when an null (or invalid) semver version is passed
             return False
+
+    @keyword
+    def Compare_Strings(self,version,target,compare):
```

Comment:
If you don't need to compare the python version (see comment below) I think this function should be removed. Also, I think the name could be improved to be more specific

If you need to keep python checks in your test, I would suggest that you add these methods to Helpers.py: 
- semver_is_gte  (self, version, target):       Returns True if the SemVer version >= target
   - This is the existing gte function renamed to semver_is_gte. You'll have to update the places where it's used  (HighAvailability.robot, InstalledOperators.robot and JupyterHubSpawner.robot)
- semver_is_equal  (self, version, target):   Returns True if the SemVer version == target
  -  When implementing this, use VersionInfo from the semver library (see method gte as example)
- semver_is_lte  (self, version, target):        Returns True if the SemVer version <=  target
  -  When implementing this, use VersionInfo from the semver library (see method gte as example)

Reply:
Updated.

Reply:
Updated.